fetch/fetchRequest.js

```diff
@@ -0,0 +1,29 @@
+/*!
+ * Copyright (c) 2018-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+require('es6-promise').polyfill();
+require('isomorphic-fetch');
+
+function fetchRequest(method, url, args) {
+  var f = fetch(url, {
+    method: method,
+    headers: args.headers,
+    body: JSON.stringify(args.data),
```

Comment:
For now, would like to leave like this because don't want to change the current logic: https://github.com/okta/okta-auth-js/blob/master/reqwest/reqwestRequest.js. We can tackle this tech debt in a later PR for the entire okta-auth-js. But good point 👍